src/bot/commands/chs.ts

```diff
@@ -0,0 +1,33 @@
+import { Args, Command } from "@sapphire/framework";
+import type { Message } from "discord.js";
+
+export class ChooseCommand extends Command {
+    public constructor(context: Command.Context, options: Command.Options) {
+        super(context, {
+            ...options,
+            name: "chs",
+            aliases: ["choose", "pilih", "mending"],
+            quotes: [],
+            description: "Choose from random items. Separated with whitespace(s) or a comma (with optional trailing spaces).",
+        });
+    }
+
+    public async messageRun(message: Message, args: Args) {
+        const r = /[\s, ]+/gi;
+        let allItems: string[] = [];
+        const responses = ["", "Hmm, ", "*refuses to elaborate further* ", "I don't know, prolly ", "It's obvious, "];
+        try {
+            const stringArgs = (await args.rest("string"));
+            allItems = stringArgs.split(r);
+            const sample =
+                allItems[Math.floor(Math.random() * allItems.length)]!;
+            const finalresp = responses[Math.floor(Math.random() * responses.length)]!.concat(sample);
+            await message.channel.send(finalresp);
```

Comment:
Instead of `concat` why not regular + operator or template string (e.g. `${response]\n${sample}`)?
Also mungkin tambahin newline di responses baris 18 biar ga langsung nyambung ama endingnya.

Reply:
aight imma try smthn